frontend/src/utils/ercTokens.tsx

```diff
@@ -10,6 +10,8 @@ export const ERC_SYMBOLS = {
   [eUSDTokenAddress]: 'eUSD',
   '0xeC76FbFD75481839e456C4cb2cd23cda813f19B1':
     'geUSD' /* goerli testnet token */,
```

Comment:
The `goerli` is staying here that way legacy testnet transactions that happened on georli will show up as geUSD in the UI, correct.

Reply:
I left this for a transition in case the auditor-db hasn't been reset. Once the auditor-db is reset, this will no longer be  helpful and can be removed in a later PR